dev/archery/archery/utils/command.py

```diff
@@ -25,7 +25,7 @@
 
 def default_bin(name, default):
     assert(default)
-    env_name = "ARCHERY_%s_BIN".format(default.upper())
+    env_name = "ARCHERY_{0}_BIN".format(default.upper())
```

Comment:
@fsaintjacques Here is a non-style issue caught by flake8.

Reply:
(note this is with the latest version of `pycodestyle`)

Reply:
What's the issue, deprecated?

Reply:
No, `%s` isn't recognized by `.format`. IOW it's a bug here.